src/components/Label/Label.tsx

```diff
@@ -4,11 +4,18 @@ import cx from 'classnames';
 type LabelProps = {
   className?: string;
   htmlFor: string;
+  isDisabled?: boolean;
   text: string;
 };
 
-export const FormLabel = ({ className, htmlFor, text }: LabelProps) => {
+export const FormLabel = ({
+  className,
+  htmlFor,
+  isDisabled,
+  text
+}: LabelProps) => {
   const classNames = cx('cc-label', {
+    'cc-label--disabled': isDisabled,
```

Comment:
I think our convention for "state" selectors is to use a new selector, e.g. `.is-disabled` (rather than `cc-label--disabled`).

@creido-welly to confirm.